### ISSUE TITLE
Mouse shooting bug

### ISSUE THREAD

**Issue by xDreamCoding** [Maescool/Catacomb-Snatch#272]
sometimes u can hold our left mousebutton but it won't shoot i know there are several issues with the mouse implementation atm just wanted to point that out.

Okay while reproducing the bug i found another issue, it will just keep shooting altough no button or mouse is pressed.

LoL keeps shootin in the background while writing this...

can even walk while autoshoot and tab in and out of the game.

not even pressing regular keyboard shoot buttons will end it.

tl;dr: Mouse shooting buggs on quick doubleclicking
